display_information/update-graphics/src/main/java/com/esri/samples/update_graphics/UpdateGraphicsSample.java

```diff
@@ -260,10 +262,14 @@ private void createGraphics() {
     }
   }
 
-  private void enableUI(boolean enable) {
-    updateDescriptionButton.setDisable(enable);
-    updateLocationButton.setDisable(enable);
-    symbolBox.setDisable(enable);
+  /**
+   * Disables the visibility of the UI controls
```

Comment:
Missing a period from this javadoc 
```suggestion
   * Disables the visibility of the UI controls.
```